-----------------------------------------------------------------
Revision: eae06ba9f36c975ca944d7b246354b6a90073df9
Ancestor: c62d41eb896d55cf80abc8073cae63eeab0db492
Author: datallah@pidgin.im
Date: 2007-07-30T22:36:20
Branch: im.pidgin.pidgin

Modified files:
        libpurple/protocols/jabber/google.c

ChangeLog: 

Fix a stupid crash (that I introduced) that occurs when parsing gmail notifications. Fixes #2323

============================================================
--- libpurple/protocols/jabber/google.c	a775804dde0650d9915fdbe431a94b7cae8354b8
+++ libpurple/protocols/jabber/google.c	3b098e3137be8a73b8f0844428f917b6d1a10815
@@ -119,8 +119,8 @@ jabber_gmail_parse(JabberStream *js, xml
 	g_free(to_name);
 	g_free(tos);
 	g_free(froms);
-	for (; i >= 0; i--)
-		g_free(subjects[i]);
+	for (; i > 0; i--)
+		g_free(subjects[i - 1]);
 	g_free(subjects);
 	g_free(urls);
 
